chipy_org/settings.py

```diff
@@ -362,3 +363,11 @@ def env_list(key, defaults=None, delimiter=","):
 
 
 STATICFILES_STORAGE = "whitenoise.storage.CompressedManifestStaticFilesStorage"
+
+
+# django-rq settings
+RQ_QUEUES = {
+    "default": {"URL": os.getenv("REDISTOGO_URL", "redis://redis:6379/0"), "DEFAULT_TIMEOUT": 500,},
+    "high": {"URL": os.getenv("REDISTOGO_URL", "redis://redis:6379/0"), "DEFAULT_TIMEOUT": 500,},
+    "low": {"URL": os.getenv("REDISTOGO_URL", "redis://redis:6379/0"), "DEFAULT_TIMEOUT": 500,},
```

Comment:
This feels like an odd name for this environment variable. Is this documented in the code somewhere as being a default or convention?

Reply:
Yes, these environment variables are required by the django-rq package.